internal/turso/organizations.go

```diff
@@ -161,3 +173,24 @@ func (c *OrganizationsClient) MembersURL(suffix string) (string, error) {
 	}
 	return "/v1/organizations/" + c.client.org + "/members" + suffix, nil
 }
+
+func unsetOrganization() error {
+	settings, err := settings.ReadSettings()
+	if err != nil {
+		return err
+	}
+	return settings.SetOrganization("")
+}
+
+func isNotMemberErr(status int, org string) bool {
+	if status == http.StatusForbidden && org != "" && unsetOrganization() == nil {
+		return true
+	}
+	return false
+}
+
+func notMemberErr(org string) error {
+	msg := fmt.Sprintf("you are not part of organization %s. ", internal.Emph(org))
+	msg += fmt.Sprintf("%s configured to use your personal organization.", internal.Emph("turso"))
```

Comment:
I'm not sure I understand this.  This suggests to me that a change of org selection automatically happened in all cases where this function `notMemberErr` is called.

Reply:
That is true. But after we do the change we need to communicate it to the user somehow, and that is why we return this error.